app/models/pot.rb

```diff
@@ -0,0 +1,6 @@
+class Pot < ApplicationRecord
+  belongs_to :owner, class_name: 'User', foreign_key: 'user_id'
+  has_many :gifts
+  has_many :users, through: :users_pot
+  validates :name, presence: true
+end
```

Comment:
I think we need to validate an end_date?

Reply:
we can set a default start and end date instead of validation.
i think for creating a name should be enough  because you are not sure when to finish it from the bat maybe

Reply:
Ok, so if they leave it blank we'd put it as +10days or something?
Start date can be the date it is created?